File: Lab Assignment 18/Task 5/java/Car.java
/**
 * Task 5: Java Car Class - Final Version with Javadoc
 */
public class CarFinal {
    String brand;   // Car manufacturer
    String model;   // Car model name
    int year;       // Manufacturing year

    /**
     * Constructor to initialize Car attributes
     */
    public CarFinal(String brand, String model, int year) {
        this.brand = brand;
        this.model = model;
        this.year = year;
    }

    /**
     * Display car details in required format
     */
    public void displayDetails() {
        System.out.println("Car Details:");
        System.out.println("Brand: " + brand);
        System.out.println("Model: " + model);
        System.out.println("Year: " + year);
    }

    public static void main(String[] args) {
        CarFinal car = new CarFinal("Toyota", "Corolla", 2020);
        car.displayDetails();
    }
}

File: Lab Assignment 18 a/Task 5/Python/car.py
"""
Task 5: Python Car Class - Final Version with Comments
"""
class Car:
    def __init__(self, brand, model, year):
        """Initialize Car object with brand, model, and year attributes"""
        self.brand = brand      # Car manufacturer
        self.model = model      # Car model name
        self.year = year        # Manufacturing year (integer)

    def display_details(self):
        """Display formatted car details"""
        print("Car Details:")
        print(f"Brand: {self.brand}")
        print(f"Model: {self.model}")
        print(f"Year: {self.year}")

if __name__ == "__main__":
    # Create and test object
    toyota = Car("Toyota", "Corolla", 2020)
    toyota.display_details()
